src/components/Tag/index.tsx

```diff
@@ -19,7 +19,7 @@ export class Tag extends PureComponent<TagProps> {
     return (
       <Container data-element-name={dataElementName} className={className}>
         <Text>{label || value}</Text>
-        {disabled !== false && (
+        {disabled !== true && (
```

Comment:
```suggestion
        {!disabled && (
```
이렇게 하는게 더 잘 읽힐 것 같아요

Reply:
undefined인 경우가 있어서 ..!! === false로 바꿀게욥

Reply:
아~~~~ 오케이